### ISSUE TITLE
[Phoenix] pressing kMode in-game causes KnightOS/Ti-84 to crash

### ISSUE THREAD

**Issue by MaxLeiter** [KnightOS/KnightOS#334]
not really important, just a small bug. Only way to fix is by replacing battery

**Comment by SirCmpwn** [comment]
MODE is pause. Press it again to unpause. Also, you can probably reboot by pressing R+ON without a battery pull.

**Comment by MaxLeiter** [comment]
I don't know why I used freeze in the title; causes the calculator to crash*

**Comment by SirCmpwn** [comment]
Crash? Like the screen turns off?

**Comment by MaxLeiter** [comment]
Yes, and if you've shot bullets it just causes the screen to... corrupt?

**Comment by MaxLeiter** [comment]
https://sr.ht/poRc.jpg <- example of what happens to Castle
